example/hello-world-cos/README.md

```diff
@@ -9,19 +9,12 @@ gcloud compute images list \
     --no-standard-images
 ```
 
-Here is how to use the debian base:
+Here is an example command:
```

Comment:
Shouldn"t these examples occur in docs/further.md?

Reply:
I prefer the examples notes for each of the examples/* to be alongside the Snakefile. The reason is because the user is likely to be sitting in that directory and will easily find it.

Reply:
ok